lib/generators/react_on_rails/base_generator.rb

```diff
@@ -6,7 +6,7 @@
 module ReactOnRails
   module Generators
     class BaseGenerator < Rails::Generators::Base # rubocop:disable Metrics/ClassLength
-      hide!
+      Rails::Generators.hide_namespace self.namespace
```

Comment:
Yep! In both 4.2.1 and 3.2.13!

http://apidock.com/rails/v3.2.13/Rails/Generators/hide_namespace/class

http://apidock.com/rails/v4.2.1/Rails/Generators/hide_namespace/class